modules/core/src/effects/mask/utils.ts

```diff
@@ -93,7 +93,7 @@ export function getMaskViewport({
   width -= padding * 2;
   height -= padding * 2;
 
-  if (viewport instanceof WebMercatorViewport) {
+  if (viewport.isGeospatial) {
```

Comment:
`isGeospatial` is also true for `GlobeView` and I suspect that this won't work for non-mercator projections

Reply:
It doesn't, although it isn't far off - the rendered mask is stretched. I've added a warning and updated the docs to make clear that `GlobeView` currently doesn't support masking in order to land this PR